### ISSUE TITLE
how to run StuttgartApplication

### ISSUE THREAD

**Issue by kainagel** [matsim-scenarios/matsim-stuttgart#34]
I spent about 1 hour trying to run `StuttgartApplication`.  I could not figure it out from the information that was provided.  Could you please provide more information :-).  Thanks ...

**Comment by Janekdererste** [comment]
Sorry, I didn't document anything, so far. I will get to this soon. You should be able to run the scenario with
```
java -jar stuttgart.jar run --config path/to/your config.xml --config:plans.inputPlansFile=path/to/your/population.xml.gz
```
Since I only recently added an application there also is a traditional run class which should be runnable with something like: 
```
java -cp path/to/your/jarfile org.matsim.stuttgart.run.RunStuttgart path/to/your/config.xml --config:plans.inputPlansFile path/to/your/population.xml.gz
```
The included config file doesn't have the correct file paths since those files are located in the shared-svn at the moment. You can find them in the shared svn:
```
https://svn.vsp.tu-berlin.de/repos/shared-svn/projects/matsim-stuttgart/stuttgart-v2.0/input/
```
Unfortunately, those files are not up to date, because I was not finished with the calibration yet. This is something I can do as soon as I have left edge land.

**Comment by flschnepf** [comment]
Hello, is it possible to get access to the input files? i would like to run the scenario

**Comment by flschnepf** [comment]
Hello, is there a way to get access to the population file or to run the scenario without a population file?

**Comment by Janekdererste** [comment]
Yes here you go https://svn.vsp.tu-berlin.de/repos/public-svn/matsim/scenarios/countries/de/stuttgart/
